schedule/staging/ext4@uefi-staging.yaml

```diff
@@ -3,27 +3,43 @@ name:           ext4@uefi-staging
 description:    >
   Test for ext4 filesystem.
 vars:
-  FILESYSTEM: ext4
-  FORMAT_DASD: install
+  YUI_REST_API: 1
+conditional_schedule:
+  access_beta_distribution:
+    BETA:
+      1:
+        - installation/access_beta_distribution
 schedule:
   - installation/bootloader_start
-  - installation/welcome
-  - installation/scc_registration
-  - installation/addon_products_sle
-  - installation/system_role
-  - installation/partitioning
-  - installation/partitioning_filesystem
-  - installation/partitioning_finish
-  - installation/installer_timezone
-  - installation/hostname_inst
-  - installation/user_settings
-  - installation/user_settings_root
+  - installation/setup_libyui
+  - '{{access_beta_distribution}}'
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_scc
+  - installation/module_registration/register_module_desktop
+  - installation/add_on_product/skip_install_addons
+  - installation/system_role/select_role_text_mode
```

Comment:
should be this one:
tests/installation/system_role/accept_selected_role_text_mode.pm